infrastructure/validators/onboarding/run-validator/secure-node.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Secure Your Validator
+description: TODO
+---
+
+Validators in a Proof of Stake network are responsible for keeping the network in consensus and verifying state transitions. As the number of validators is limited, validators in the set have the responsibility to be online and faithfully execute their tasks.
+
+This primarily means that validators:
+
+- Must be high availability
+- Must have infrastructure that protects the validator's signing keys so that an attacker cannot take control and commit [slash-able behavior]()
+
+## High Availability
+
+High availability set-ups that involve redundant validator nodes may seem attractive at first. However, they can be *very dangerous* if they aren't set up perfectly. The reason for this is that the session keys used by a validator should always be isolated to just a single node. Replicating session keys across multiple nodes could lead to equivocation [slashes]() or parachain validity slashes which can make you lose **100% of your staked funds**.
+
+The good news is that 100% uptime of your validator isn't really needed, as it has some buffer within eras to go offline for a little while and upgrade. For this reason, it is advised that you only attempt a high availability set-up if *you're confident you know exactly what you're doing.*
+
+The proper handling of session keys is one of the biggest ways a validator can keep their node safe.
+
+## Key Management
+
+See the [Polkadot Keys guide]() for more information on keys. The keys
+that are of primary concern for validator infrastructure are the Session keys. These keys sign messages related to consensus and parachains. Although Session keys are _not_ account keys and therefore cannot transfer funds, an attacker could use them to commit slash-able behavior.
+
+Session keys are generated inside the node via RPC call. See the [How to Validate guide]() for instructions on setting Session keys. These should be generated and kept within your client. When you generate new Session keys, you must submit an extrinsic (a Session certificate) from your staking proxy key telling the chain your new Session keys.
```

Comment:
I'd like to add a reference to the RPC call that you mentioned here. If I'm not mistaken, it is the following:

```suggestion
Session keys are generated inside the node via [RPC call](https://paritytech.github.io/polkadot-sdk/master/sc_rpc_api/author/trait.AuthorApiClient.html#method.insert_key){target=\_blank}. See the [How to Validate guide]() for instructions on setting Session keys. These should be generated and kept within your client. When you generate new Session keys, you must submit an extrinsic (a Session certificate) from your staking proxy key telling the chain your new Session keys.
```